src/object/obj_internal.h

```diff
@@ -113,10 +113,26 @@ enum_anchor_copy(daos_anchor_t *dst, daos_anchor_t *src)
 }
 
 extern struct dss_module_key obj_module_key;
+enum obj_profile_op {
+	OBJ_UPDATE_PREP = 0,
```

Comment:
maybe with OBJ_PF_  prefix is better, to differentiate with other normal obj OP

Reply:
ok